.github/workflows/packaging-release.yml

```diff
@@ -0,0 +1,57 @@
+name: Packaging and Release
+
+on:
+  workflow_dispatch: # Allow to manually run the workflow.
+  push:
+    branches:
+    - main
+
+jobs:
+  choco-building:
+    runs-on: windows-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+      - name: Choco packaging
+        uses: crazy-max/ghaction-chocolatey@v3
+        with:
+          args: pack Choco\vmagent\vmagent.nuspec
+      - uses: actions/upload-artifact@v3
+        with:
+          name: vmagent.nupkg
+          path: vmagent.*.nupkg
+
+  choco-publishing:
+    needs: choco-building
+    runs-on: windows-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+      - uses: actions/download-artifact@v3
+        with:
+          name: vmagent.nupkg
+      - name: Choco push
+        uses: crazy-max/ghaction-chocolatey@v3
+        with:
+          args: push --source https://push.chocolatey.org/ -k ${{ secrets.CHOCO_API_KEY }}
+
+  release:
+    needs: choco-building
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+      - uses: actions/download-artifact@v3
+        with:
+          name: vmagent.nupkg
+      - name: Set version
+        run: |
+          echo "VM_VERSION=$(cat VERSION)" >> $GITHUB_ENV
+      - name: Rename choco package for release
+        run: |
+          mv vmagent.*.nupkg vmagent.nupkg
+      - name: Release
+        uses: softprops/action-gh-release@v1
```

Comment:
Tried the "request change" functionality. Cannot resolve manually. Now we need to push something new :shrug:

Reply:
It can handle using the same release again. It will delete all the release packages and then use the newly built one.

Other then that you would have a manual "Create tag" workflow which I want to avoid as best as possible.
My Dream is: There is a PR from the Bot that there is a new version. I click on merge. Everything is done automatically.